README.md

```diff
@@ -476,18 +477,19 @@ Find sectors that have a certain coverage from origin (or to destination) and ge
 * results: List[SectorsResult] - The results list of postcode sectors.
 
 #### Example:
+
 ```python
 from datetime import datetime
 
-from traveltimepy import Coordinates, PublicTransport, TravelTimeSdk, ZonesProperty
+from traveltimepy import Coordinates, PublicTransport, TravelTimeSdk, PostcodesProperty
 
 sdk = TravelTimeSdk('YOUR_APP_ID', 'YOUR_APP_KEY')
 
-results = sdk.sectors(
-    coordinates=[Coordinates(lat=51.507609, lng=-0.128315)],
-    departure_time=datetime.now(),
-    transportation=PublicTransport(),
-    properties=[ZonesProperty.TRAVEL_TIME_REACHABLE, ZonesProperty.TRAVEL_TIME_ALL]
+results = sdk.postcodes_sectors(
+  coordinates=[Coordinates(lat=51.507609, lng=-0.128315)],
+  departure_time=datetime.now(),
+  transportation=PublicTransport(),
+  properties=[PostcodesProperty.TRAVEL_TIME_REACHABLE, PostcodesProperty.TRAVEL_TIME_ALL]
```

Comment:
I'd stay with ZonesProperty cause postcodes have their own properties which are different from these ones

Reply:
Maybe `PostcodesZoneProperty`?

Reply:
just rename this please and we can merge it